[2023-08-01 11:27:58,087] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: s3_to_BigQuery.stations_data_to_BQ manual__2023-08-01T11:27:56.377712+00:00 [queued]>
[2023-08-01 11:27:58,094] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: s3_to_BigQuery.stations_data_to_BQ manual__2023-08-01T11:27:56.377712+00:00 [queued]>
[2023-08-01 11:27:58,094] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-08-01 11:27:58,094] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2023-08-01 11:27:58,094] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-08-01 11:27:58,115] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): stations_data_to_BQ> on 2023-08-01 11:27:56.377712+00:00
[2023-08-01 11:27:58,125] {standard_task_runner.py:52} INFO - Started process 4096 to run task
[2023-08-01 11:27:58,131] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 's3_to_BigQuery', 'stations_data_to_BQ', 'manual__2023-08-01T11:27:56.377712+00:00', '--job-id', '180', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmp3y3mm9rj', '--error-file', '/tmp/tmpk5g6l4s5']
[2023-08-01 11:27:58,133] {standard_task_runner.py:80} INFO - Job 180: Subtask stations_data_to_BQ
[2023-08-01 11:27:58,215] {task_command.py:370} INFO - Running <TaskInstance: s3_to_BigQuery.stations_data_to_BQ manual__2023-08-01T11:27:56.377712+00:00 [running]> on host a6bbb8f7ba0b
[2023-08-01 11:27:58,326] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=s3_to_BigQuery
AIRFLOW_CTX_TASK_ID=stations_data_to_BQ
AIRFLOW_CTX_EXECUTION_DATE=2023-08-01T11:27:56.377712+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-08-01T11:27:56.377712+00:00
[2023-08-01 11:27:58,327] {logging_mixin.py:115} INFO - 
 heeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeyy
[2023-08-01 11:29:03,971] {logging_mixin.py:115} INFO - 
✅ DataFrame created ! ✅
[2023-08-01 11:29:03,972] {logging_mixin.py:115} INFO - Error: [Errno 2] No such file or directory: '/home/andre/NguyenTrdc/lime-project/raw_data/lime-project-key.json'
[2023-08-01 11:29:04,285] {python.py:173} INFO - Done. Returned value was: None
[2023-08-01 11:29:04,297] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=s3_to_BigQuery, task_id=stations_data_to_BQ, execution_date=20230801T112756, start_date=20230801T112758, end_date=20230801T112904
[2023-08-01 11:29:04,655] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-08-01 11:29:04,952] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
